backend/tests/test_stub.py

```diff
@@ -69,13 +69,19 @@ async def test_books_add_endpoint_save_book_metadata(book_dict):
     assert response.status_code == 201
     assert response.headers.get("Content-Type") == "application/json"
 
-    _metadata = await BookMetadata.objects.all(book=book_dict["id"])
-    for _ in _metadata:
-        assert _.value == book_dict["metadata"][_.name]
+    metadata_ = await BookMetadata.objects.fields("id", "metadata").all(
+        book=book_dict["id"]
+    )
+
+    assert len(metadata_) == len(book_dict["metadata"])
+    for _ in metadata_:
```

Comment:
never use `_` if you intend to use it. `_` is a convention for stuff that is not used.

Reply:
@rgaudin does this also apply to list/set comprehension?

Reply:
absolutely. You use it bellow (`_.name`). If you use it, use a properly named variable like `item` here.

`_` is used when a return values is not to be used.

```py
for _ in range(10):
    print(".")  # don't need the number returned by range here
```